### ISSUE TITLE
Add TYPE definition inside TimelineChart class

### ISSUE THREAD

**Issue by grillorafael** [commodityvectors/d3-timeline#14]

**Comment by grillorafael** [comment]
@pocketjoso how do you suggest doing that?

**Comment by pocketjoso** [comment]
Actually I would export them separately, perhaps:
```
export const TYPE = {
  POINT: Symbol(),
  INTERVAL: Symbol()
}
```

.. but that would not be backwards compatible with what you've got currently, so if so you should keep the current assignment too, at least until you release a major version of your library.

**Comment by grillorafael** [comment]
Yes. In this case it is better to be in another major release. Let's keep as `TimelineChart.TYPE` for now. What It might be useful is to add something like `Symbol('point')` to make it easier to read when debugging.
